1_BayesIntuition.ipynb

```diff
@@ -0,0 +1,893 @@
+{
```

Comment:
<p>* instead of "algorithm called Markov Chain Monte Carlo" maybe "an algorithm of the Markov Chain Monte Carlo family of algorithms" or maybe "a numerical method" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/canyon289/pyladies-bayes/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>